### ISSUE TITLE
Accessibility workflow not running correctly

### ISSUE THREAD

**Issue by James-Makela** [NM-TAFE/dip-programming-prj-advanced-gui-awesome#10]
![image](https://github.com/NM-TAFE/dip-programming-prj-advanced-gui-awesome/assets/112157557/aa046d3c-af31-47db-b7eb-50d644072ec9)

Accessibility workflow run was failing anyway, but now gets stuck since commit [f2fbb8e](https://github.com/NM-TAFE/dip-programming-prj-advanced-gui-awesome/commit/f2fbb8e647ed2a3899f27e4b96bc47f2b2b25504)

This appears to be caused by the addition of a new module - `flask_socketio` used in app.py. 

Updating requirements.txt should fix this.
